Doc/library/exceptions.rst

```diff
@@ -409,14 +409,16 @@ The following exceptions are the exceptions that are usually raised.
 
    .. versionadded:: 3.5
 
-.. exception:: SyntaxError
+.. exception:: SyntaxError(message, details)
 
    Raised when the parser encounters a syntax error.  This may occur in an
-   :keyword:`import` statement, in a call to the built-in functions :func:`exec`
+   :keyword:`import` statement, in a call to the built-in functions
```

Comment:
It can also happen with some interfaces in the ast module like `ast.parse` but I don't think we wanna have an exhaustive list here.

Reply:
Agreed.   I was thinking 'functions in builtins'.  Further change in this paragraph would be a different issue.  (I would prefer a different ordering, but not enough to write a PR.)

What do you think of the other additions, which are the main focus?  Is this ready to go with your correction?

Reply:
Yeah, everything looks good! Thanks a lot for doing this!